### PR TITLE
POST /api/reviews/:review_id/comments endpoint

### PR DESCRIPTION
**Pull request by stormytuna**
Branch name is wrong, `git branch -m post-comment-by-review-id` didnt fix it :|

### CODE REVIEW

**Comment by stormytuna** [comment]
not sure why the commits from the other branch are on here

File: models/models.comments.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+		return comments;
+	});
+};
+
+exports.insertCommentByReviewId = (comment, reviewId) => {
+	const query = `
+    INSERT INTO comments (body, author, review_id, votes, created_at)
+    VALUES
+      ($1, $2, $3, 0, $4)
```
Comment (k-huggs) [inline models/models.comments.js:27]: votes and created at are defaulted in your seed function

File: models/models.comments.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    RETURNING *;
+  `;
+	const params = [comment.body, comment.username, reviewId, new Date(Date.now())];
```
Comment (k-huggs) [inline models/models.comments.js:30]: Have a look at what the database is expected to return

File: __tests__/app.test.js [hunk 3]
```diff
@@ -114,3 +114,114 @@ describe("GET /api/reviews/:review_id", () => {
...
+						comment_id: expect.any(Number),
+						votes: expect.any(Number),
+						created_at: expect.any(String),
+						author: expect.any(String),
+						body: expect.any(String),
+						review_id: expect.any(Number)
+					});
+				});
+			});
+	});
```
Comment (k-huggs) [inline __tests__/app.test.js:137]: still need to add test for if you're getting back an empty array

File: __tests__/app.test.js [hunk 4]
```diff
@@ -114,3 +114,114 @@ describe("GET /api/reviews/:review_id", () => {
...
+			username: "mallionaire",
+			body: "very cool :)"
+		};
+		return request(app)
+			.post("/api/reviews/4/comments")
+			.send(newComment)
+			.expect(201)
+			.then(({ body }) => {
+				const { comment } = body;
+				expect(comment).toMatchObject({
```
Comment (k-huggs) [inline __tests__/app.test.js:172]: The trello specifies we only want back the username and body. Not the whole comment object

File: __tests__/app.test.js [hunk 5]
```diff
@@ -114,3 +114,114 @@ describe("GET /api/reviews/:review_id", () => {
...
+					comment_id: expect.any(Number),
+					votes: 0,
+					created_at: expect.any(String),
+					author: "mallionaire",
+					body: "very cool :)",
+					review_id: 4
+				});
+			});
+	});
```
Comment (k-huggs) [inline __tests__/app.test.js:181]: need an additional 404 for if the username doesn't exist

**Review by k-huggs — COMMENTED** [review]
Really good code thus far, I think you've branched from a branch potentially.
Have a look at some of the comments theres a couple of amendments 🙂

File: __tests__/app.test.js (hunk 3 above)
Comment (stormytuna) [reply __tests__/app.test.js:137]: Already done in 7e7b5d65686f164fec9f2a61e8c16ccbd356d38a
